api/taskcluster/webhook.go

```diff
@@ -160,9 +160,14 @@ func handleStatusEvent(ctx context.Context, payload []byte) (bool, error) {
 	// The default timeout is 5s, not enough for the receiver to download the reports.
 	slowCtx, cancel := context.WithTimeout(ctx, resultsReceiverTimeout)
 	defer cancel()
-	var labels []string
+	labels := mapset.NewSet()
 	if status.IsOnMaster() {
```

Comment:
Not related to this change, but I'm curious if master on a fork will match this condition? (That might result in confusion.)

Reply:
Yes, it would, but it'd have to be a PR from master to... something. And I think the logic here being questionable will probably carry over to the config for taskcluster choosing master vs other stuff.

Reply:
master-to-master PRs are pretty common, it's what happens when people just clone their fork and make some changes.

Reply:
Yes, but the task names are different in prs, so we already identify them as pr_head / pr_base by that stage.